### ISSUE TITLE
SQLite + Poem can't be built due to: use of undeclared crate or module `argonautica`

### ISSUE THREAD

**Issue by zelid** [Wulf/create-rust-app#84]
SQLite + Poem can't be built due to `use of undeclared crate or module argonautica`

```
~/W/f/c/demo-1-sqlite-poem>  cargo build
  ...
   Compiling demo-1-sqlite-poem v0.1.0 (/Users/alex/Workspace/frontend-2023/create-rust-app-examples/demo-1-sqlite-poem)
   Compiling dotenv v0.15.0
   Compiling hyper v0.14.23
   Compiling dsync v0.0.5
   Compiling jsonwebtoken v8.2.0
   Compiling tracing-subscriber v0.3.16
   Compiling async-graphql v3.0.38
   Compiling diesel_migrations v2.0.0
   Compiling aws-smithy-http v0.38.0
   Compiling hyper-rustls v0.22.1
   Compiling poem v1.3.50
   Compiling aws-smithy-http-tower v0.38.0
   Compiling aws-sigv4 v0.8.0
   Compiling aws-smithy-client v0.38.0
   Compiling aws-endpoint v0.8.0
   Compiling aws-sig-auth v0.8.0
   Compiling aws-http v0.8.0
   Compiling aws-sdk-sso v0.8.0
   Compiling aws-sdk-sts v0.8.0
   Compiling aws-sdk-s3 v0.8.0
   Compiling aws-config v0.8.0
   Compiling async-graphql-poem v3.0.38
   Compiling create-rust-app v8.1.1
error[E0433]: failed to resolve: use of undeclared crate or module `argonautica`
 --> /Users/alex/.cargo/registry/src/github.com-1ecc6299db9ec823/create-rust-app-8.1.1/src/dev/endpoints/service_poem.rs:1:5
  |
1 | use argonautica::config::Version::_0x10;
  |     ^^^^^^^^^^^ use of undeclared crate or module `argonautica`

For more information about this error, try `rustc --explain E0433`.
error: could not compile `create-rust-app` due to previous error
```

**Comment by AnthonyMichaelTDM** [comment]
Pretty sure I fixed this a few weeks ago, fixes should ship in the next release

@Wulf

**Comment by zelid** [comment]
> 

That's what happens in the latest available version `create-rust-app v8.1.1`.
Is the fix in some newer (maybe -beta) version?

**Comment by Wulf** [comment]
@zelid  thanks for reporting this. Unfortunately I’ll have to delay the release which fixes this issue until I cleanup the commit history in main. I don’t exactly know what happened and I tried to investigate but I’ll just have to force push local history I have and try to recreate previous merges.

**Comment by Wulf** [comment]
Thanks for reporting this :raised_hands:, it should be fixed in 9.0.0. Feel free to reopen as needed!~
